packages/cli/src/utils/directory.ts

```diff
@@ -1,69 +1,47 @@
 import path from 'path'
 
-// build folder
-export const tmpFolderName = '.faststore'
+export const withBasePath = (basepath: string) => {
```

Comment:
@gvc considering that among the files that are calling `withBasePath`, with the exception of the tests, are passing the same argument to your closure (`process.cwd()`). Wouldn't it fit if we consider `process.cwd()` as the default possible path, if no arguments are called?

Meaning that instead of

```js
const basePath = process.cwd()
const { tmpDir } = withBasePath(basePath)
```

We would do

```js
const { tmpDir } = withBasePath()

// directory.ts
export const withBasePath = (basepath?: string = process.cwd()) => { ... }
```

This would also positively resonate to how `generate f()` is handling `basePath` there, as it does a similar approach.

Reply:
I can see making that `process.cwd()` the default value, but I wouldn't omit in the places I'm calling because of the next PR that will likely finish this feature 😬

Reply:
Basically, the default will cease to be useful on the next PR :)

Reply:
@gvc understood! thanks for clarifying :D